integration_tests/exemplars_otel/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.prometheus</groupId>
+    <artifactId>integration_tests</artifactId>
+    <version>0.10.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>exemplars_otel</artifactId>
+  <name>Integration Test - Exemplars with OpenTelemetry</name>
+
+  <properties>
+    <otel.version>1.1.0</otel.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.prometheus</groupId>
+      <artifactId>simpleclient</artifactId>
+      <version>${project.version}</version>
+      <exclusions>
+        <!-- uncomment the following to verify manually that simpleclient still works if the tracers are excluded -->
+        <!--
+        <exclusion>
+          <groupId>io.prometheus</groupId>
+          <artifactId>tracer_common</artifactId>
+        </exclusion>
+        -->
+        <exclusion>
+          <groupId>io.prometheus</groupId>
+          <artifactId>tracer_otel_agent</artifactId>
+        </exclusion>
+        <!--
+        <exclusion>
+          <groupId>io.prometheus</groupId>
+          <artifactId>tracer_otel</artifactId>
+        </exclusion>
+        -->
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-api</artifactId>
+      <version>${otel.version}</version>
```

Comment:
I added the maven wrapper script so that users have a defined version of maven (specified in `.mvn/wrapper/maven-wrapper.properties`) if they call `./mvnw` instead of `mvn`.

Reply:
My comment stands, you can't assume that users are using maven.